Content.Shared/Zombies/ZombieEvents.cs

```diff
@@ -0,0 +1,25 @@
+using Content.Shared.Actions;
+
+namespace Content.Shared.Zombies;
+
+/// <summary>
+///     Event that is broadcast whenever an entity is zombified.
+///     Used by the zombie gamemode to track total infections.
+/// </summary>
+public sealed class EntityZombifiedEvent : EventArgs
```

Comment:
```suggestion
public sealed class EntityZombifiedEvent : EntityEventArgs
```

Or if you want to be based make it a struct ref event.